### ISSUE TITLE
API Changes / Segregation

### ISSUE THREAD

**Issue by jreback** [pandas-dev/pandas2#66]
This will serve as a master issue to record API changes / testing that needs segregation in pandas.

Copy-on-write can help here:
- [ ] view propgation: https://github.com/pandas-dev/pandas/issues/14953

**Comment by wesm** [comment]
As a matter of process, the way this should probably work is:

- Segregate unit tests subject to breakage in upstream pandas-dev/pandas
- Rebase pandas2 on upstream branch. 
- Write new unit tests that port those "bad" tests to new tests that reflect the changed behavior 

what do you think?

**Comment by jreback** [comment]
yep, should have the test runner exclude them for pandas2 (and add separate tests that *only*) apply to pandas 2.

so prob need something like

pandas/tests # existing tests
pandas/tests_changing_in_pandas2
pandas/tests_pandas2

with the sub-structure the same as
pandas/tests

**Comment by jorisvandenbossche** [comment]
From the dev meeting: idea is instead of physically segregating the tests is to leave them at the same place but mark them as "may fail on pandas 2.0"
